commcare_connect/opportunity/views.py

```diff
@@ -1632,3 +1653,118 @@ def user_visit_details(request, org_slug, opp_id, pk):
             is_program_manager=is_program_manager,
         ),
     )
+
+
+def opportunity_worker(request, org_slug=None, opp_id=None):
+    opp = get_opportunity_or_404(opp_id, org_slug)
+    base_kwargs = {"org_slug": org_slug, "opp_id": opp_id}
+    visit_export_form = VisitExportForm()
+    export_form = PaymentExportForm()
+
+    path = [
+        {"title": "Opportunities", "url": reverse("opportunity:list", args=(org_slug,))},
+        {"title": opp.name, "url": reverse("opportunity:detail", args=(org_slug, opp_id))},
+        {"title": "Workers", "url": reverse("opportunity:worker_list", args=(org_slug, opp_id))},
+    ]
+
+    raw_qs = request.GET.urlencode()
+    query = f"?{raw_qs}" if raw_qs else ""
+
+    tabs = [
+        {
+            "key": "workers",
+            "label": "Workers",
+            "url": reverse("opportunity:worker_table", kwargs=base_kwargs) + query,
+            "trigger": "loadWorkers",
+        },
+        {
+            "key": "learn",
+            "label": "Learn",
+            "url": reverse("opportunity:learn_table", kwargs=base_kwargs) + query,
+            "trigger": "loadLearn",
+        },
+        {
+            "key": "delivery",
+            "label": "Delivery",
+            "url": reverse("opportunity:delivery_table", kwargs=base_kwargs) + query,
+            "trigger": "loadDelivery",
+        },
+        {
+            "key": "payments",
+            "label": "Payments",
+            "url": reverse("opportunity:payments_table", kwargs=base_kwargs) + query,
+            "trigger": "loadPayments",
+        },
+    ]
+
+    return render(
+        request,
+        "tailwind/pages/opportunity_worker.html",
+        {
+            "opportunity": opp,
+            "tabs": tabs,
+            "visit_export_form": visit_export_form,
+            "export_form": export_form,
```

Comment:
todo: name more clearly to know what form is exporting

Reply:
Looks like this form gets used for multiple export modals (users and payments) with just different endpoint.